facts/4.0/almalinux-8-x86_64.facts

```diff
@@ -459,37 +420,13 @@
     }
   },
   "path": "/home/vagrant/vendor/bundler/ruby/2.5.0/bin:/opt/puppetlabs/puppet/bin:/opt/puppetlabs/bin:/usr/local/sbin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin",
-  "physicalprocessorcount": 1,
-  "processor0": "Intel(R) Core(TM) i7-9800X CPU @ 3.80GHz",
-  "processorcount": 1,
-  "processors": {
```

Comment:
processors is als deprecated?

Reply:
so `processor\d` is deprecated, the `processors` hash isn't: https://www.puppet.com/docs/puppet/7/core_facts.html

I think this was removed by accident. good catch!

Reply:
Wow!  Good catch!